kod/object/active/holder/nomoveon/battler/monster/towns/kocatwn/kcbowmak.kod

```diff
@@ -88,6 +92,9 @@ messages:
          send(self,@SetMood,#new_mood=piMood + 2);
       }
 
+      Send(poOwner,@SomeoneSaid,#what=self,#string=KocatanBowMaker_welcome_rsc,
```

Comment:
Everyone in the room is going to hear this every time anyone walks in?  I thought we recently changed a bunch of NPCs to at least send this only to the person who enters.  Do the same here?